Packs/HatchingTriage/Integrations/HatchingTriage/HatchingTriage.py

```diff
@@ -190,9 +190,14 @@ def get_report_triage(client: Client, **args) -> CommandResults:
     """
     sample_id = args.get("sample_id")
     task_id = args.get("task_id")
+    if not task_id.startswith("behavioral"):
+        return_error(
+            "Only behavioral reports can be retrieved with this command. "
+            "Task ID must be 'behavioral' followed by a number. "
+            "E.G: 'behavioral1'"
+        )
```

Comment:
So I think the description in the yml file should change to something like `Name of the task for the sample (e.g. behavioral1, behavioral2, etc)` and not `Name of the task for the sample (e.g. behavioral1, static1, etc)` as it's described now